python/docs/tutorial.rst

```diff
@@ -249,17 +248,17 @@ Now let's create a simple client to test this service out.
    :linenos:
 
 As well as sending requests, we need to be able to get back the
-responses. We create a receiver for that (see line 14), but we don't
+responses. We create a receiver for that (see line 15), but we don't
 specify an address, we set the dynamic option which tells the broker
-we are connected to to create a temporary address over which we can
+we are connected to a temporary address over which we can
```

Comment:
For this particular sentence, the original is correct, though a little unclear.  Simplified, the sense is: the option tells the broker to create an address.